omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -126,8 +126,8 @@ Using conda (preferred)
 
 #. Create virtual environments named ``omeropy`` and ``omeroweb``::
 
-    $ conda create -n omeropy -c ome python=3.6 zeroc-ice36-python omero-py
-    $ conda create -n omeroweb -c ome python=3.6 zeroc-ice36-python omero-web
+    $ conda create -n omeropy -c conda-forge python=3.6 zeroc-ice36-python omero-py
+    $ conda create -n omeroweb -c conda-forge python=3.6 zeroc-ice36-python omero-web
```

Comment:
There is no https://anaconda.org/conda-forge/omero-web. Should this be pip installed for now?